jemalloc-sys/Cargo.toml

```diff
@@ -32,7 +33,7 @@ libc = { version = "^0.2.8", default-features = false }
 cc = "^1.0.13"
 
 [features]
-default = ["stats", "background_threads_runtime_support"]
```

Comment:
Considering we are releasing a new minor version, "stats" is removed from the default features to make it easier to control features.

/cc @girlbossceo

Reply:
Is there any strong reason blocking us from just increasing the patch version?

Reply:
Now jemalloc-sys will disable jemalloc stats explicitly if stats feature is not enabled. This may break crates that rely on jemalloc stats but not enable it explicitly. To avoid breaking compatibility (though it's a bug), stats feature was added to the default feature set. But it still breaks those crates that disable the default features. It's more clear to upgrade to a new minor version and ask everyone relies on stats feature to enable it explicitly.